WebPhapp/backend/server.js

```diff
@@ -1,9 +1,34 @@
 const express = require('express');
 const path = require('path');
 var fs = require("fs");
+var conn = require('./connections.js') // private file not under VC.
 
 const app = express();
 
+// establish a connection to the remote MySQL DB
+if(conn.MySQL){
+    var mysql = require('mysql')
+    var connection = mysql.createConnection(conn.MySQL);
+}
+
+// Given a list of drugIDs, looks up drug names in the MySQL DB.
+// Args: drugIDs list <(int or string)>
+// Returns: Promise
+function getDrugNamesFromIDs(drugIDs){
```

Comment:
Likely open to an SQL injection because of the values are not being escaped. HUGE security issue to just concatenate strings. I personally use https://www.npmjs.com/package/mysql2-promise to deal with this on my personal website.

Reply:
Yeah, I can do the initial setup on the prepared statements. Accept as is then I'll make this one of my tasks for the week. 

Still bad practice to have non-prepared statemented queries. It may not be an instant issue but it could be a 'second order' issue.